rebar.config

```diff
@@ -132,10 +132,8 @@
         {relx, [
             {sys_config, "./config/docker_rosetta_testnet.config"},
             {dev_mode, false},
-            {include_erts, false},
-            {overlay, [
-                {copy, "priv/genesis_testnet", "update/genesis"}
-            ]}
+            {include_erts, false}
```

Comment:
Why did you not do this for the other targets too?

Reply:
It wasn't obvious to me how we should make rebar3 pull a file from an http source for other build targets - I guess there's `rebar3_raw_resource` which basically does that?

Reply:
I'm not understanding.. why can you remove it here but not in other profiles?

Reply:
@jadeallenx bump